src/ess/loki/general.py

```diff
@@ -61,6 +61,7 @@ def default_parameters() -> dict:
 
 
 def _convert_to_tof(da: sc.DataArray) -> sc.DataArray:
+    da = da.copy(deep=False)
```

Comment:
We do not want to deep-copy since that is costly. We need a shallow copy that also shallow copies the data array representing the bin contents. You can probably use what you implemented in https://github.com/scipp/scipp/pull/3518.

Reply:
I see what you mean